config.js

```diff
@@ -0,0 +1,15 @@
+require('dotenv').config()
```

Comment:
Not needed, please remove. 

`process.env` is defined as long as `main.js` imports dotenv before `config.js`, so you don't have to reimport.